server/config.js

```diff
@@ -0,0 +1,6 @@
+const boardGameGeek = new URL('https://gametableproxy.herokuapp.com');

```

Comment:
You shouldn't need to redefine both the host and protocol as they are already defined by what is input in the constructor